### ISSUE TITLE
Loading eror

### ISSUE THREAD

**Issue by CoderBaitu** [bui/taiko-web#458]
### Discussed in https://github.com/bui/taiko-web/discussions/452

<div type='discussions-op-text'>

<sup>Originally posted by **Lollolopop** March 16, 2022</sup>

![Screenshot_2022-03-17-08-22-33-958_com android chrome](https://user-images.githubusercontent.com/79087322/158712844-a4b7dfcd-2daa-4522-94ae-5334d6f0a6ca.jpg)
</div>

Hello! I am also having an issue with loading taiko-web on Chrome as well! In my case, I'm running it on MacOS instead of a phone, however. No matter how many times I reload, it doesn't seem to fix itself properly. Keep in mind that it was working perfectly fine before this occurred.

<img width="1082" alt="Screen Shot 2022-03-23 at 12 21 18 AM" src="https://user-images.githubusercontent.com/76254647/159623651-a18f1b62-2c15-456c-b62d-a4b3dda6ea6f.png">

**Comment by KatieFrogs** [comment]
Can you post a screenshot of the errors that get printed into the developer console
Here is a guide on how to access it: https://balsamiq.com/support/faqs/browserconsole/
When you open the console, refresh the page and screenshot it

**Comment by KatieFrogs** [comment]
Yes thank you, can you try clearing the browser cache to see if that solves the issue
Guide for Safari: https://clear-my-cache.com/apple-mac-os/safari.html
